### ISSUE TITLE
Impact appraisals

### ISSUE THREAD

**Issue by andylolz** [pwyf/2018-technical-consultation#32]
## Description

Pre-project impact appraisals explain the totality of positive and negative, primary and secondary effects expected to be produced by a development intervention.

This is only expected if the activity is in the implementation, completion or post-completion phase.

This is not expected if the sector code is administrative costs (91010).

## Proposed test

```python
For each current activity,
if `activity-status/@code` is one of (2, 3, 4)
and `sector/@code` is not 91010
then `document-link/category[@code="A01"]` should be present
```

**Comment by breidertmt** [comment]
Every project does not have this.

**Comment by francescafo** [comment]
I agree. It is often not included in project-related documents. Organisations should not be penalised for this.

**Comment by AndieVaughn** [comment]
To echo the comments above, it is unrealistic that 100% of programs and projects need an impact appraisal.  A full score should be perhaps 50% and not 100% in order to account for the fact that some transactions will not need this (beyond already excluded administrative costs).

**Comment by ToonVB** [comment]
We would like to reiterate what we indicated under "objectives":

> We think that it's neither useful to require objectives in the case of (assessed or voluntary) core-support to multilateral organisations (B02).
Moreover: in the case of core-support to Belgian NGO's (B01), in which we co-fund programs, we (will) require them to publish their activities in IATI format themselves, as UK and NL are already doing as well.
So we will only publish the commitment and disbursement, leaving them to publish activities, objectives, results and so on.
The same applies for disbursements type of aid B03 and B04.
And more generally, in fact objectives and results are only formulated in the case of C01 - projects.

**Comment by andylolz** [comment]
Thanks for the comments here.

We’re in agreement that activities with aid type B02 (core contributions to multilateral institutions) should be ignored by this test.

We will also modify the name of this indicator to be clear that we’re expecting a pre-project impact appraisal here.
